yarn.lock

```diff
@@ -3424,6 +3424,11 @@
   dependencies:
     ethers "^5.0.2"
 
+"@typechain/ethers-v5@^6.0.1":
```

Comment:
I think you didn't have the latest arbos checked out when you last ran `yarn`. I don't believe there should be any `yarn.lock` diff in this PR

Reply:
Added